tests/containers/bci_prepare.pm

```diff
@@ -41,6 +41,8 @@ sub packages_to_install {
         my $version = "$version.$sp";
         push @packages, 'python3-devel';
         if ($version eq "12.5") {
```

Comment:
What about 12-SP4? Pretty sure if `jq` is missing on 12-SP5 then 12-SP4 (and 12-SP3) will also complain.

Reply:
yeah, I had to check myself till which version we are using in the tests :)

Reply:
We don't run this on 12-SP4. Or do we?